_posts/2022-09-14-download-tool.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: Jennifer Chen
+title: New Data Download Tool
+---
+
+In the last year, we have added a ton of new data and many exciting new features. One such feature is the new Data Download tool that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
+
+![](/assets/images/posts/download_tool.png)
```

Comment:
nit: add some alt-text inside the [] to improve accessibility

Reply:
done